utils/remote_api_console.py

```diff
@@ -3,29 +3,43 @@
 import getpass
 import logging
 import sys
+import os
 
 logging.basicConfig(level=logging.INFO)
 
-base_path = "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine"
+# base_path = "/Applications/GoogleAppEngineLauncher.app/Contents/Resources/GoogleAppEngine-default.bundle/Contents/Resources/google_appengine"
+base_path = "C:\\Users\\Audrey\\AppData\\Local\\Google\\Cloud SDK\\google-cloud-sdk\\platform\\google_appengine"
```

Comment:
I know this is a work in progress, but these backslashes make this look like a horrible Windows-specific hack.  You might want to `from pathlib import Path` and work with Python's new(ish) `Path` class - it should provide a nice cross-platform implementation.